charts/posthog/values.yaml

```diff
@@ -251,7 +266,7 @@ ingress:
   hostname:
   gcp:
     # -- Specifies the name of the global IP address resource to be associated with the google clb
-    ip_name:
+    ip_name: "posthog"
```

Comment:
Why? Removed it on purpose.

Reply:
Because we get an error if this isn't specified and I wasn't sure if that needs to be something our users need to specify or if it's fine for this to be `"posthog"`. I could also try to look into the failure we see from `helm install --debug --dry-run posthog charts/posthog`, but this seemed easier.

Reply:
Issue being that while install will work with this specified, it will not work - the instance will not be accessible unless there actually is an IP with that name created. 

See https://posthog.com/docs/deployment/deploy-gke-clickhouse#step-by-step-installation

Reply:
discussed offline - keeping this, will make the documentation be clear that the name must match with the IP name they set (& that the default is posthog)